pkg/subctl/cmd/logs.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/submariner-io/submariner-operator/pkg/discovery/network"
+)
+
+func getConnectivityPodsLogs(clientSet kubernetes.Interface) (bool, error){
+	pods, err := network.FindPods(clientSet, v1.ListOptions{LabelSelector: "app=submariner-gateway"})
+
+	if err != nil {
+		fmt.Println("error getting pods")
+		return false, err
+	}
+	for _, pod := range pods.Items {
+		_, err := getPodLogs(clientSet, pod.Name)
```

Comment:
I think it would be useful to `QueueSuccessMessage`/`QueueFailureMessage` for each pod. For success, include the pod name and output file name, eg "_Log for pod %q successfully persisted to file %q_".